src/max/sum/TwoDArray.java

```diff
@@ -1,21 +1,22 @@
 package max.sum;
 
-public class twoDArray {
+public class TwoDArray {
     int[][] rectangle;
     int size;
```

Comment:
make rectangle and size private?

Reply:
A good point. @shweinfeld was the author of this code